### ISSUE TITLE
Exempt feature checker for com.ibm.wsspi.webcontainer.WCCustomProperties

### ISSUE THREAD

**Issue by pmd1nh** [OpenLiberty/open-liberty#5910]
This class is frequently updated.  It is better to exempt it from all the feature checker test.

**Comment by pnicolucci** [comment]
I don't think we can exempt it since it is SPI. However we need to figure out why the tooling isn't working as expected. For instance see: https://github.com/OpenLiberty/open-liberty/issues/3553 There are two PRs of interest, https://github.com/OpenLiberty/open-liberty/pull/5826 works fine because the updates to WCCustomProperties were removed, however what we really want is the changes in https://github.com/OpenLiberty/open-liberty/pull/3557. The build fails and tells us to update a bundle version and then fails again when we do that and says we should not have. We have gotten around this previously by adding excludes but I think this needs to be investigated and fixed  properly.
